src/relayFeeCalculator/chain-queries/zksync.ts

```diff
@@ -4,12 +4,13 @@ import { providers } from "ethers";
 import { TOKEN_SYMBOLS_MAP, CHAIN_IDs } from "../../constants";
 import { DEFAULT_SIMULATED_RELAYER_ADDRESS } from "./baseQuery";
 import QueryBase from "./baseQuery";
+import { getDeployedAddress } from "@across-protocol/contracts-v2";
```

Comment:
This will break the FE unfortunately... Can you import from 
```js
import { getDeployedAddress } from "../../utils/DeploymentUtils";
```
instead? Like for example in https://github.com/across-protocol/sdk-v2/blob/7ef7316168d3e463d7b20a9236c410abfa685e77/src/relayFeeCalculator/chain-queries/arbitrum.ts#L1